Nodejs/Product/Nodejs/Repl/NodejsReplEvaluator.cs

```diff
@@ -39,6 +39,20 @@ public NodejsReplEvaluator(INodejsReplSite site)
             this._site = site;
         }
 
+        private string nodeExePath;
+
+        public string NodeExePath
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(this.nodeExePath))
```

Comment:
Should this be a straight null check?  Wouldn't empty string indicate that it had already run (and failed)?

Reply:
Yes, but when a new project is loaded we want to try again, maybe they configured the Node location there correct.